CBOnlineIos/Presentation/LibraryViewController.swift

```diff
@@ -0,0 +1,19 @@
+
+
+import UIKit
+
+class LibraryViewController: UIViewController {
+
+    @IBOutlet weak var NavigationBar: UIView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+                NavigationBar.layer.cornerRadius = 20
```

Comment:
Instead of re-writing code, again and again, use a base view controller or either use an extension function

Reply:
I have made a common parent class